client/src/views/landing.vue

```diff
@@ -76,25 +86,28 @@ export default class Landing extends Vue {
 }
 </script>
 
-<style lang="sass" scoped>
-#landing-container
-  margin: 0
-  padding: 0
+<style lang="scss" scoped>
+@import '@/assets/scss/theme.scss';
 
-.landing-content-container
-  min-width: 940px
-  max-width: 1140px
-
-#upper-row
-  background: url('../assets/images/analyze-name-bg.jpg') no-repeat bottom
-  background-size: cover
-  color: white
-  min-height: 700px
-
-.beta-tag
-  top: -1rem
-  color: $gold
-  font-size: 19px
-  font-weight: bold
-  text-transform: uppercase
+#landing-container {
+  margin: 0;
+  padding: 0;
```

Comment:
This will be easier to clean up in the future if you use `ma-0 pa-0` in its class instead.

Reply:
ah good call!

Reply:
Yeah, I can see where we might want to redefine, say, containers, to not have negative margins or whatever, and clean up a bunch of the HTML.
(Probably not as an explicit ticket but as cleanup in other tickets.)

Reply:
Absolutely, agree 100%. Updated!

Reply:
Those class helpers have changed my life 😆